pod/pod-api-public.yaml

```diff
@@ -6801,6 +6801,10 @@ definitions:
         type: boolean
         description: |
           File extension allowed for external scope
+      scope_xpod:
+        type: boolean
+        description: |
+          File extension allowed to be received from another POD
```

Comment:
you need to add x-since with the version of sbe this is implemented in

Reply:
> you need to add x-since with the version of sbe this is implemented in

ok i see, but which version should i put? Seems like SBE now uses another naming, like 24.1 we have to day for latest realises. Should i put "x-since: '24.1'" then ?

Reply:
you can put 24.2, we'll update it if the name changes

Reply:
done